app/support/reservations/relay_support.rb

```diff
@@ -3,6 +3,7 @@ module Reservations::RelaySupport
   def can_switch_instrument_on?(check_off = true)
     return false if canceled?
     return false unless product.relay # is relay controlled
+    return false if product.offline?
```

Comment:
Should this be checking the shared schedule? I know we haven't necessarily settled that question yet, but other PRs have been assuming we do share the schedule.